### ISSUE TITLE
VideoStreaming

### ISSUE THREAD

**Issue by LaryLee** [FREEZX/fq777-954-chrome-app#8]
Have a look over here: https://github.com/Otacon/wifi_china_drone_controller
This guy managed to stream video. It is working very well, as long, as you keep the wifi chip cool. If you don't, the video may start to lag.

**Comment by FREEZX** [comment]
I have the video stream data running, my issue is with decoding it and displaying it in chrome.
It's here: https://github.com/FREEZX/fq777-954-chrome-app/blob/master/js/main.js#L206

**Comment by LaryLee** [comment]
Is the stream data valid? Did you try to write the data to a file stream and open it with ffplay or vlc? What does ffprobe say to the data?

Perhaps a workaround could be to provide a socket for ffplay and pass the received data through to it.

**Comment by FREEZX** [comment]
Yep, i have saved the data and it's playing in ffmpeg. Here it is:
https://github.com/FREEZX/fq777-954-chrome-app/blob/master/out.mp4
I tried Broadway but didn't have enough time to dig too deep.
